test/libsolidity/syntaxTests/using/using_global_library_for_interface.sol

```diff
@@ -0,0 +1,6 @@
+interface I {}
+using L for I global;
+library L {
+}
```

Comment:
Again, can you add a positive test? This would be allowed without `global`, right?